pulse/pulse/views.py

```diff
@@ -95,4 +101,4 @@ def report_feed():
 
     @app.errorhandler(404)
     def page_not_found(e):
-      return render_template('404.html'), 404
+      return render_template('/en/404.html'), 404
```

Comment:
Hey, instead of duplicating all of the views to have one that points to `en/.,.` and `fr/...`, what do you think about adding a `prefix` param to `register`, and calling it with `prefix`="en" and `prefix`="fr" in app initialization area?

This doesn't have to have to be for this PR, just a thought I had reading this

Reply:
def interested in a more elegant solution! Will address it in a different PR, as I add in the French pages